presto-docs/src/main/sphinx/installation/cli.rst

```diff
@@ -14,9 +14,11 @@ Download :maven_download:`cli`.
 
 Rename the JAR file to ``presto`` with the following command: 
 
+(Replace ``*`` in the example with the version number of the downloaded jar file)
```

Comment:
I think it should work as-is, since in the command-line the clob will just match the exact file name.  i.e. I think we can remove this comment.